config.example.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
+    name: '/spaces',
+    authProviders: {
+        matrix: {
+            baseUrl: 'https://matrix.org',
+            allowCustomHomeserver: true,
+        },
+        etherpad: {
+            path: '/write',
+            baseUrl: 'https://pad.riseup.net/p',
+            myPads: {
+                api: 'http://etherpad.localhost/mypads/api',
+                spacesGroupName: '/spaces',
+            },
+        },
+    },
+    contextRootSpaceRoomId: '!gB.....Ewlvdq:matrix.org',
+    account: {
+        allowAddingNewEmails: true,
+    },
+    chat: {
+        pathToElement: '//localhost/element',
+    },
+};
```

Comment:
So basically -this- file is going to be the new **medienhaus-spaces** config file that we ask developers and administrators to look at when wanting to configure our app.